src/test/java/com/karankumar/bookproject/ui/goal/ReadingGoalViewTest.java

```diff
@@ -118,22 +120,22 @@ void testSetGoalButtonTextUpdatesWhenGoalUpdates() {
     }
 
     @Test
-    void testTargetMetMessageNotShownWhenGoalNotMet() {
+    void notShownTargetMetMessageWhenGoalNotMet() {
         String expected = ReadingGoalView.TARGET_MET;
         String actual = goalView.calculateProgress(GOAL_TARGET, GOAL_TARGET - 1);
         assertNotEquals(expected, actual);
     }
 
     @Test
-    void testTargetMetMessageShownWhenGoalMet() {
+    void shownTargetMetMessageWhenGoalMet() {
```

Comment:
Typo here: this should be `show`, not `shown` (it made sense before but not here)